packages/shared/src/keys.ts

```diff
@@ -39,7 +39,10 @@ export function parsePublishableKey(
   key = key || '';
 
   if (!key || !isPublishableKey(key)) {
-    if (options.fatal) {
+    if (options.fatal && !key) {
+      throw new Error('Publishable key is missing.');
```

Comment:
Can we be more specific with the error here? What are remediation steps to address?